### PR TITLE
IGNITE-13970 ThinkClient service, application, simple test

### PR DESCRIPTION
**Pull request by eadha**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

**Comment by anton-vinogradov** [comment]
@emvdovets, 
First of all, you have Travis failed. Please always check this before asking for a review.
Also, I see a lot of copy-pasted duplication, please solve this and avoid it in the future.

File: modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/thin_client_test/ThinClientCachePut.java [hunk 1]
```diff
@@ -48,10 +48,10 @@ protected void run(JsonNode jsonNode) throws Exception {
         ClientCacheConfiguration cfg = new ClientCacheConfiguration();
         cfg.setName(cacheName);
 
-        System.out.println(">>> Cache requested [cacheName=" + cacheName + ']');
-        System.out.println(">>>                 [cacheMode=" + cacheMode + ']');
-        System.out.println(">>>                 [cacheAtomcityMode=" + cacheAtomcityMode + ']');
-        System.out.println(">>>                 [backups=" + backups + ']');
+        System.out.println(">>> Cache requested  [cacheName=" + cacheName + ']');
```
Comment (anton-vinogradov) [inline modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/thin_client_test/ThinClientCachePut.java:51]: We should never use System.out.println, use a logger instead.
We should also avoid logging using some messages for a single event, this approach does not work in the case of concurrency.

File: modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/utils/SpringThinClientConfig.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                        <value>ducker01:10800</value>
+                        <value>ducker02:10800</value>
+                        <value>ducker03:10800</value>
+                        <value>ducker04:10800</value>
+                        <value>ducker05:10800</value>
+                        <value>ducker06:10800</value>
+                        <value>ducker07:10800</value>
+                        <value>ducker08:10800</value>
+                        <value>ducker09:10800</value>
+                        <value>ducker10:10800</value>
```
Comment (anton-vinogradov) [inline modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/utils/SpringThinClientConfig.xml:19]: will this work or read the environment?
looks like some hardcode or autogenerated code.

**Comment by dspavlov** [comment]
This PR is associated with a resolved ticket. Closing the PR, please reopen it if you still need it.

**Pull request closed without merging**